Server/bin/OpenClubList.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
+
+function ReadClubFile()
+{
+    var jsonString = fs.readFileSync("./data/datafromclub.json", "utf8");
+    var object = JSON.parse(jsonString);
+    return object.clubs;
+}
+
+module.exports = {ReadClubFile};
```

Comment:
This file is not a script or executable. Move it to another directory than `bin/`.